socorro/unittest/middleware/test_middleware_app.py

```diff
@@ -530,11 +528,20 @@ def _insert_products(self):
         """)
         self.conn.commit()
 
-    def _setup_config_manager(self, extra_value_source=None):
+    def _setup_config_manager(
+        self,
+        extra_value_source=None,
+        services_list=None
+    ):
         if extra_value_source is None:
             extra_value_source = {}
         extra_value_source['web_server.wsgi_server_class'] = MyWSGIServer
         mock_logging = mock.Mock()
+        if services_list:
+            middleware_app.MiddlewareApp.SERVICES_LIST = services_list
+        else:
+            # the global list
+            middleware_app.MiddlewareApp.SERVICES_LIST = middleware_app.SERVICES_LIST
```

Comment:
Before this change, the list of services was stuck. It was and only was what was written down in `middleware_app.py`. It could not be changed on a per-test-basis. No, making it an attribute of the class makes it possible to test things on the class with a varying list of services.